spec/index.html

```diff
@@ -2686,8 +2690,246 @@ <h3>Selective Disclosure Schemes</h3>
+      harming the ability of the recipient to verify the authenticity of those selected statements.</p>
+    
+    <p>The output of the canonicalization algorithm described in this specification, may leak partial
+      information about undisclosed statements and help the adversary correlate the original and disclosed datasets.</p>
+    
+    <section id="privacy-considerations-leakage-labeling">
+      <h4>Possible Leakage via Canonical Labeling</h4>
+    
+      <p>If a dataset contains at least two blank nodes, the canonical labeling can be exploited to guess the undisclosed
+        quad in the dataset.</p>
+      
+      <p>For example, let us assume we have the following dataset to be signed. (Note: this person is fictitious, prepared
+        only to make this example work.)</p>
+
+      <pre id="ex-pc-leakage-labeling-original-dataset" class="example" data-transform="updateExample" title="Original Dataset">
+        <!--
+          _:b0 <http://schema.org/address> _:b1 .
+          _:b0 <http://schema.org/familyName> "Jarrett" .
+          _:b0 <http://schema.org/gender> "Female" .  # gender === Female
+          _:b0 <http://schema.org/givenName> "Ali" .
+          _:b0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
+          _:b1 <http://schema.org/addressCountry> "United States" .
+          _:b1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/PostalAddress> .
+        -->
+      </pre>
+
+      <p>Using <a href="#canon-algorithm" class="sectionRef"></a>, we can obtain the <a>serialized canonical form</a> of the
+          <a>normalized dataset</a>, where all the blank nodes are serialized using the canonical labels.</p>
+
+      <pre id="ex-pc-leakage-labeling-normalized-dataset" class="example" data-transform="updateExample" title="Normalized Dataset">
+        <!--
+          _:c14n0 <http://schema.org/addressCountry> "United States" .
+          _:c14n0 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/PostalAddress> .
+          _:c14n1 <http://schema.org/address> _:c14n0 .
+          _:c14n1 <http://schema.org/familyName> "Jarrett" .
+          _:c14n1 <http://schema.org/gender> "Female" .  # gender === Female
+          _:c14n1 <http://schema.org/givenName> "Ali" .
+          _:c14n1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://schema.org/Person> .
+        -->
+      </pre>
+
+      <p>The signer can generate a signature for the dataset by first hashing each statement and then signing them
+        using a multi-message digital signature scheme like BBS+. The resulting dataset with signature is passed to the
+        holder, who can control whether or not to disclose each statement while maintaining their verifiability.</p>
+      
+      <p>Let us say that the holder wants to show her attributes except for `gender` to a verifier. Then the holder should
```

Comment:
Actually using pronouns such as "his" and "her" ends up getting flagged, so prefer gender neutral terms.

```suggestion
      <p>Let us say that the holder wants to show their attributes except for `gender` to a verifier. Then the holder should
```